### PR TITLE
Add proxy support [#16]

### LINKED ISSUE #16: Add proxy support to Python SDK
## How to Reproduce
1. Look for proxy in the SDK codebase

## What Should Happen
1. Proxy can be set in `ApiContext`

## What Happens
1. No proxy can be set in `ApiContext`

### PR DESCRIPTION
**Pull request by dnl-blkv**
Fixes #16

### CODE REVIEW

**Comment by dnl-blkv** [comment]
@OGKevin deserialisation is fixed! 👍

**Comment by OGKevin** [comment]
@dnl-blkv My tests are failing while creating new context. 

```
File "/Users/khellemun/GitHub/bunq/sdk_python/bunq/sdk/client.py", line 105, in _request
    proxies={self._FIELD_PROXY_HTTPS: self._api_context.proxy_url}
AttributeError: 'ApiContext' object has no attribute 'proxy_url'
```

https://github.com/bunq/sdk_python/blob/40627c5ca46c9b00fe2afecd1b6655b022849339/tests/bunq_test.py#L40

What could be causing this ?

**Comment by OGKevin** [comment]
Hmm weird just did a clean checkout again merged dev and tests are passing 🤔

File: bunq/sdk/context.py [hunk 1]
```diff
@@ -57,12 +58,13 @@ class ApiContext(object):
     _PATH_API_CONTEXT_DEFAULT = 'bunq.conf'
 
     def __init__(self, environment_type, api_key, device_description,
-                 permitted_ips=None):
+                 permitted_ips=None, proxy_url=None):
```
Comment (OGKevin) [inline bunq/sdk/context.py]: This function has a lot of arguments, why not reformat it to:
```python
def __init__(self, environment_type, api_key,
                   device_description,  permitted_ips=None, proxy_url=None):
```

3 args on a row or:

```python
    def __init__(
            self,
            environment_type,
            api_key,
            device_description,
            permitted_ips=None,
            proxy_url=None):
```
1 arg per line ?

Comment (dnl-blkv) [inline bunq/sdk/context.py]: 1) Not PEP-8
2) Takes too much space

This one is OK, we do it everywhere :)

Comment (OGKevin) [inline bunq/sdk/context.py]: ack

**Pull request merged**